src/main/java/ludogorie_soft/reservations_platform_api/controller/PropertyController.java

```diff
@@ -0,0 +1,37 @@
+package ludogorie_soft.reservations_platform_api.controller;
+
+import lombok.RequiredArgsConstructor;
+import ludogorie_soft.reservations_platform_api.dto.PropertyRequestDto;
+import ludogorie_soft.reservations_platform_api.dto.PropertyResponseDto;
+import ludogorie_soft.reservations_platform_api.service.PropertyService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.io.FileNotFoundException;
+
+@RestController
+@RequestMapping("/api/property")
```

Comment:
better use plural form `/api/properties`
https://medium.com/@nadinCodeHat/rest-api-naming-conventions-and-best-practices-1c4e781eb6a5

Reply:
Done!